src/components/Avatar/README.md

```diff
@@ -210,11 +214,14 @@ LANDING_BLOCK-->
 
 ## CSS API
 
-| Name                          | Description             |
-| :---------------------------- | :---------------------- |
-| `--g-avatar-size`             | Size (width and height) |
-| `--g-avatar-background-color` | Background color        |
-| `--g-avatar-border-color`     | Border color            |
-| `--g-avatar-color`            | Icon and text color     |
-| `--g-avatar-font-size`        | Text font size          |
-| `--g-avatar-line-height`      | Text line height        |
+| Name                            | Description             |
+| :------------------------------ | :---------------------- |
+| `--g-avatar-size`               | Size (width and height) |
+| `--g-avatar-border-width`       | Border width            |
+| `--g-avatar-inner-border-width` | Inner border width      |
+| `--g-avatar-border-color`       | Border color            |
+| `--g-avatar-background-color`   | Background color        |
+| `--g-avatar-color`              | Icon and text color     |
```

Comment:
Let's name it `--g-avatar-text-color` similar to other components

Reply:
Just `color` because this color is used for icon and text both

Reply:
FIXED